components/director/README.md

```diff
@@ -12,3 +12,19 @@ go run cmd/main.go
 ```
 
 The GraphQL API playground is available at `localhost:3000`.
+
+## Configuration
```

Comment:
Consider adding instruction how to run director locally, without creating Docker image

Reply:
Readme updated.